everyvoice/cli.py

```diff
@@ -14,6 +14,9 @@
 from everyvoice.model.aligner.wav2vec2aligner.aligner.cli import (
     align_single as ctc_segment,
```

Comment:
If you don't want to keep this called `align_single`, maybe call it `ctc_segment_align` instead of just `ctc_segment`, to reduce the confusion of what this function is.